src/sass/components/govcy-header.scss

```diff
@@ -2,57 +2,50 @@
 // This file contains all styles related to the header of the site/application.
 // -----------------------------------------------------------------------------
 
-$govcy-sub-menu-color: #577589;
-// $govcy-sub-menu-color:$govcy-info;
-$padding: 0.8rem;
-/* menu */
-/* menu icon */
-/* menu btn */
+// Mobile first styles for header section starts here
+
 .govcy-header {
   padding: 0;
-  font-family: $govcy-font-family;
-  background-color: $govcy-primary;
-  box-shadow: 0.0625rem 0.0625rem 0.25rem 0 rgba(0, 0, 0, 0.1);
-  width: 100%;
-  z-index: 3;
-  max-height: 12.8rem;
-  ul {
-    margin: 0;
-    padding: 0;
-    list-style: none;
-    overflow: hidden;
-    background-color: $govcy-sub-menu-color;
-    li {
-      a {
-        display: block;
-        padding: $padding !important;
-        color: $govcy-white;
-        border-bottom: 0.125rem solid #31576f;
-        text-decoration: none;
-        margin-left: 0.625rem;
-        margin-right: 0.625rem;
-        &:hover {
-          background-color: $govcy-btn-primary-hover;
-        }
-      }
-      &:last-child a {
-        border-bottom: none;
-      }
-    }
+  min-width: 350px;
+
+  //Govcy logo
+  .govcy-logo {
+    float: left;
+    height: 62px;
+    content: $govcy-logo;
   }
 
-  .govcy-header-menu-btn {
-    &:hover {
-      background-color: $govcy-btn-primary-hover;
-    }
-    display: none;
+  //Govcy service name
+  .govcy-service-name {
+    float: left;
+    margin: 16px 24px ; 
```

Comment:
20px 24px 0px 24px


Might work to get that thing aligned with the logo 🤷

Reply:
i have pull some corrections for that ben. It looks good to me know , please when you have time please review